### PR TITLE
Added supported sensors list

### PR DESCRIPTION
**Pull request by roni-kreinin**
- Warnings on broken link so that we can build the site (There are a lot of broken links right now.)
- Minor fixes as discussed in #121

### CODE REVIEW

File: docs/ros/ros/installation/controller.mdx [hunk 1]
```diff
@@ -6,24 +6,15 @@ toc_min_heading_level: 2
 toc_max_heading_level: 4
 ---
 
-Joystick controllers are used for teleoperation; they allow you to remotely drive the Jackal, whether it is a physical Jackal robot,
-or a simulated Jackal. The following instructions below detail how to pair different controllers to the Jackal’s computer; however,
+Joystick controllers are used for teleoperation; they allow you to remotely drive the robot, whether it is a physical robot,
+or a simulated robot. The following instructions below detail how to pair different controllers to the robot's computer; however,
 these instructions can also be used to pair these controllers to your own computer.
 
-### Logitech F710 Controller
```
Comment (tonybaltovski) [inline docs/ros/ros/installation/controller.mdx:13]: There might be Husky A200s with these.

File: docs/ros/ros/installation/remote_pc.mdx [hunk 2]
```diff
@@ -1,20 +1,22 @@
...
 
 ROS 2 Humble uses [Ubuntu 22.04](https://releases.ubuntu.com/jammy/) as its Tier 1 operating system. Though other operating systems are supported,
-it is highly recommended to use Ubuntu 22.04. For the remote PC, Ubuntu 22.04 Desktop should be installed.
+it is highly recommended to use Ubuntu 22.04. For the remote computer, Ubuntu 22.04 Desktop should be installed.
 
 ### Networking
 
-The remote PC should be connected to the same Wi-Fi network as the robot PC, or should be connected via ethernet. If connected over ethernet,
-set your static IP to the `192.168.131` subnet, with an IP address that won't interfere with other devices on the network, such as `192.168.131.99`.
+The remote computer should be connected to the same Wi-Fi network as the robot computer, or should be connected via ethernet. If connected over ethernet, set the static IP of the remote computer to the `192.168.131` subnet, with an IP address that won't interfere with other devices on the network, such as `192.168.131.99`.
```
Comment (tonybaltovski) [inline docs/ros/ros/installation/remote_pc.mdx]: to the `192.168.131.1/24` network,

File: docs/ros/ros/installation/controller.mdx (hunk 1 above)
Comment (civerachb-cpr) [reply docs/ros/ros/installation/controller.mdx:13]: There definitely are; a project shipped with one relatively recently (though it specifically is unlikely to get upgraded to Humble anytime soon).  All the older Kinetic/Melodic era robots that shipped with the 710 controllers may see customers upgrading, especially if they were sold to research labs or universities.
Plus robots that have a Microhard for a base-station may not have a wireless/bluetooth module on the motherboard, meaning the F710 would be the preferred controller.

**Review by tonybaltovski — APPROVED** [review]
Just some minor nits.

File: docs/ros/ros/installation/controller.mdx (hunk 1 above)
Comment (roni-kreinin) [reply docs/ros/ros/installation/controller.mdx:13]: We still support them in the robot.yaml but we are considering them deprecated now. There isn't really any setup for them anyways.

https://github.com/clearpathrobotics/cpr-documentation/pull/121#discussion_r1240145145

File: docusaurus.config.js [hunk 3]
```diff
@@ -11,7 +11,7 @@ const config = {
   title: "Clearpath Robotics Documentation",
   url: "https://docs.clearpathrobotics.com",
   baseUrl: "/",
-  onBrokenLinks: "throw",
+  onBrokenLinks: "warn",
```
Comment (jhiggins-cpr) [inline docusaurus.config.js:14]: Are we sure we want to change this? I'm ok to change it on the ros-humble branch, but I don't think we would push this change up to development.

Comment (roni-kreinin) [reply docusaurus.config.js:14]: This is temporary. I think a lot more links will break while we restructure everything. Just need to remember to revert this.

Comment (jhiggins-cpr) [reply docusaurus.config.js:14]: Ok sounds good

**Review by jhiggins-cpr — APPROVED** [review]

**Pull request merged**